scout/utils/acmg.py

```diff
@@ -303,6 +305,7 @@ def get_acmg_temperature(acmg_terms: set) -> Optional[dict]:
         (points, temperature, point_classification)
 
     """
+    LOG.warning(f"acmg_terms received:{acmg_terms}")
```

Comment:
The right terms arrive to this function:

<img width="443" alt="image" src="https://github.com/user-attachments/assets/b7ec95fe-a461-4e96-aba9-a8ef61c560c7" />